ignore/src/types.rs

```diff
@@ -301,7 +301,7 @@ const DEFAULT_TYPES: &'static [(&'static str, &'static [&'static str])] = &[
     ("vimscript", &["*.vim"]),
     ("wiki", &["*.mediawiki", "*.wiki"]),
     ("webidl", &["*.idl", "*.webidl", "*.widl"]),
-    ("xml", &["*.xml", "*.xml.dist"]),
+    ("xml", &["*.xml", "*.xml.dist", "*.dtd", "*.xsl", "*.xslt", "*.xsd", "*.xjb", "*.rng", "*.sch"]),
```

Comment:
I think these all look good to me.

Could you please use a style consistent with the surrounding code? In particular, please wrap code to 79 columns inclusive.

Thanks!

Reply:
No problem, I will update it.

Do you want me to add another commit or can i force-push the change to the existing commit (rewriting history there)?

Reply:
Force push please, if convenient, in order to keep this as a single commit. (Otherwise, I will squash this into one commit. I try to keep ripgrep's revision history very clean.)

Reply:
Done.